src/installationmanager.h

```diff
@@ -242,6 +262,9 @@ class InstallationManager : public QObject, public MOBase::IInstallationManager
   QString m_CurrentFile;
   QString m_ErrorMessage;
 
+  // List of creates files:
```

Comment:
Apart from the typo, specify what the qstring is:
map of created entries -> temporary path